xtask/src/ci.rs

```diff
@@ -211,14 +212,29 @@ fn run_wasm_checks(cmd: Option<WasmFeatureSet>) -> Result<()> {
             .run()
     };
 
+    let test_command_bot = || {
+        let _p = pushd("crates/matrix-sdk/examples/wasm_command_bot");
+
+        cmd!("rustup run stable cargo clippy --target wasm32-unknown-unknown")
+            .args(["--", "-D", "warnings", "-A", "clippy::unused-unit"])
+            .run()
```

Comment:
Not sure whether it's better, but we could use cargo's `--manifest-path` instead of `pushd`.